rpc/sosapi_client.c

```diff
@@ -1943,13 +1944,30 @@ int dsos_transaction_begin(dsos_container_t cont, struct timespec *timeout)
 	int rc = 0;
 	dsos_session_t sess = cont->sess;
 	struct timespec now;
+	struct timespec until;
```

Comment:
Can't this just be a local variable?

Reply:
Oh sorry, the diff made it look like it was part of the container structure. My bad...